test/plugins/features/movies/controller.test.js

```diff
@@ -82,15 +82,49 @@ describe('movie controller', () => {
 
   describe('retrieve movies with locations', () => {
 
+    beforeEach(async () => {
+      await Knex('locations_movies').insert(testLocMov);
+    });
+
     it('retrieves a movie with the related locations', async () => {
       const query = {};
       const movies = await Controller.list(query);
 
-      expect(movies.length).to.eql(2);
+      expect(movies.length).to.eql(3);
       expect(movies.models[0].get('name')).to.eql(testMovie1.name);
       expect(movies.models[0].relations.locations.models[0].attributes.name).to.eql(testLocation.name);
     });
 
   });
 
+  describe('add location to movie', () => {
+
+    it('adds a location to a movie', async () => {
```

Comment:
You should also add a test that checks that the call throws when trying to add to a nonexistent movie!